15314f1ddb882700dc9fabc5ca961943/update/sys_dictionary_x_illu2_illumio_illumio_label_dimension_isdeleted.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update>
+    <sys_dictionary action="INSERT_OR_UPDATE" element="isdeleted" table="x_illu2_illumio_illumio_label_dimension">
+        <active>true</active>
+        <array>false</array>
+        <attributes/>
+        <audit>false</audit>
+        <calculation><![CDATA[(function calculatedFieldValue(current) {
+
+	// Add your code here
+	return '';  // return the calculated value
+
+})(current);]]></calculation>
+        <choice/>
+        <choice_field/>
+        <choice_table/>
+        <column_label>isDeleted</column_label>
+        <comments/>
+        <create_roles/>
+        <default_value>false</default_value>
+        <defaultsort/>
+        <delete_roles/>
+        <dependent/>
+        <dependent_on_field/>
+        <display>false</display>
+        <dynamic_creation>false</dynamic_creation>
+        <dynamic_creation_script/>
+        <dynamic_default_value/>
+        <dynamic_ref_qual/>
+        <element>isdeleted</element>
+        <element_reference>false</element_reference>
+        <foreign_database/>
+        <formula/>
+        <function_definition/>
+        <function_field>false</function_field>
+        <internal_type display_value="">boolean</internal_type>
+        <mandatory>false</mandatory>
+        <max_length>40</max_length>
+        <name>x_illu2_illumio_illumio_label_dimension</name>
```

Comment:
why is the name chosen as "x_illu2_illumio...." ? i see this being used in other places too
it can be renamed to illumio_label_dimension?

Reply:
Is it possible to change the scope name to something more readable?

Reply:
To change the scope name we’d have to create a new app. Hence, we wouldn’t be able to change the scope name.